SAI/README.md

```diff
@@ -0,0 +1,21 @@
+```
+usage: sai_api_gen.py [-h] [--print-sai-lib PRINT_SAI_LIB]
+                      [--sai-git-url SAI_GIT_URL]
+                      [--ignore-tables IGNORE_TABLES]
+                      [--sai-git-branch SAI_GIT_BRANCH]
+                      filepath apiname
+
+P4 SAI API generator
+
+positional arguments:
+  filepath              Path to P4 program BMV2 JSON file
+  apiname               Name of the new SAI API
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --print-sai-lib PRINT_SAI_LIB
+  --sai-git-url SAI_GIT_URL
+  --ignore-tables IGNORE_TABLES
+                        Coma separated list of tables to ignore
+  --sai-git-branch SAI_GIT_BRANCH
+```
```

Comment:
Could you please provide an example with valid arguments? Thanks.

Reply:
Got error trying to run with just `-h`, looks like some dependencies are needed. Could you provide the appropriate commands or a script? Thanks. See below:
```
DASH-1/SAI$ ./sai_api_gen.py -h
Traceback (most recent call last):
  File "./sai_api_gen.py", line 6, in <module>
    from git import Repo
ModuleNotFoundError: No module named 'git'
```

Reply:
Added example